cl/disclosures/utils.py

```diff
@@ -42,3 +42,29 @@ def has_been_extracted(data: Dict[str, Union[str, int, list]]) -> bool:
     return FinancialDisclosure.objects.filter(
         download_filepath=url, has_been_extracted=True
     ).exists()
+
+
+def make_disclosure_data(person) -> Tuple:
+    """
+
+    :param person:
+    :return:
+    """
```

Comment:
```suggestion
def make_disclosure_data(person: Person) -> Tuple[str, str]:
    """Make a CSV of the years and the IDs of somebody's disclosures

    :param person: The Person we're making data for
    :return: A tuple that can be passed to an HTML data attribute, and which
    contains a CSV of the years of the disclosures and their associated IDs.
    """
```

Reply:
This required an import  but thank you for it none the less